### DISCUSSION TITLE
【求助】这是发生了什么？

### DISCUSSION THREAD

**Discussion by SeeleVollereid** [share121/inter-knot#2353] [其他]
![Screenshot_2024-08-16-09-42-10-01_a252b927494330cdc2c8ba3b3f952e5e](https://github.com/user-attachments/assets/919a9ad2-e522-4c09-82a4-306e14a84588)
![Screenshot_2024-08-16-09-46-04-33_a252b927494330cdc2c8ba3b3f952e5e](https://github.com/user-attachments/assets/d329094e-262d-4915-a998-703d4551bb14)

**Comment by SteveNotSet** [comment]
被封号了
